src/webapp/src/client/app/common/directives/wall/comments/comment-create/comment-create.component.js

```diff
@@ -79,9 +86,29 @@
             }
         }
 
+        function selectMention(item) {
+            vm.selectedMentions.push({id: item.id, fullName: item.label});
+
+            return `**@${item.label.replace(' ', '_')}**`;
```

Comment:
Missed that part and found about it only while testing..
Can we not use markdown syntax to highlight users? It looks a bit weird, and user can modify and remove "**" manually.
In edit form it's enough to have it plain, and during rendering the post - just parse the message and bold/emphasize it.

Reply:
Yes, I left it for the reason that user can change how mention is shown. 
User can easily see what is going to happen with his text and can remove any markdown.
However, my views on this can be wrong.

![image](https://user-images.githubusercontent.com/51354779/89758752-30dd3b80-daf1-11ea-9433-925b4ad0b2da.png)

Reply:
I think it could be not intuitive enough comparing to other apps - i.e. Slack, Jira, etc.
User mention highlighting is "hardcoded" functionality, not part of MarkDown syntax, etc.
My views could be wrong as well there, but just comparing to experience with other apps. :)

Reply:
Ok. I will hide markdown then. I also believe it's possible to add hiden markdown for hyperlink to user's profile, as we fetch user id with name aswell, if this functionality is needed.

Reply:
The link would be good, but optional. I.e. depends how much effort needed and if there are no other consequences (like impact on performance).